ts/packages/node/src/stores/LaunchDarkly.ts

```diff
@@ -0,0 +1,173 @@
+import { Config, ConfigStore, ConfigStoreQuery } from '@configu/ts';
+import axios, { Axios } from 'axios';
+import assert from 'node:assert';
+import * as fs from 'fs';
+
+export type LaunchDarklyConfigStoreConfigurations = { apitoken: string; defaultproject: string } | string;
+
+export class LaunchDarklyConfigStore extends ConfigStore {
+  private client: Axios;
+  private readonly projectKey: string;
+  private environments: string[];
```

Comment:
I don't see a reason to hold the environment in memory if we refetch them every time the class instance is used. If there are helper methods that need the environments, let's pass them to them using the latest environments that we anyway fetch at the start of every `get` or `set`.

Reply:
I mean... you're generally right.. but feels like exaggeration. it's an array of strings. in JS i think it's as memory-optimized as plain string since it's also an array of chars.
But if it's crucial to you i'll make it `getEnv..` and it will return them instead.

Reply:
Please remove from Class